proposals/0020-no-type-in-type.rst

```diff
@@ -285,7 +277,58 @@ Alternatives
    I find both arguments compelling independently, and so I withdraw support for this
    alternative. Nevertheless, I'm keeping it in the proposal in case someone wants to
    argue in support of it.
-   
+
+Implication TypeOperators to NoStarIsType
```

Comment:
This section title reads strangely. I would suggest "Have `TypeOperators` imply `NoStarIsType`" as an alternative title. (Make sure to fix the forward reference to it in the Effects section as well.)